packages/app/src/elements/item-view.ts

```diff
@@ -234,64 +237,17 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
-    private _removeExpiration() {
-        if (!this._item) {
-            return;
-        }
-
-        this._expiresEveryPeriodOfDays = undefined;
-
-        if (!this._editing) {
-            this.save();
-        }
-    }
-
-    private _renewItemExpiration(days: number) {
-        if (!this._item) {
-            return;
-        }
-
-        this._expiresEveryPeriodOfDays = days;
-
-        if (!this._editing) {
-            this.save();
-        }
-    }
-
-    private async _addExpirationDate() {
-        const promptDialog = (await getDialog("pl-prompt-dialog")) as PromptDialog;
-
-        const message = () => {
-            return html` <div class="break-words">
-                ${$l(`Please enter the number of days for this item to expire after the last update.`)}
-            </div>`;
-        };
-
-        const daysToAdd: number = await promptDialog.show({
-            title: $l("Add Expiration Time"),
-            placeholder: $l("Enter expiration in number of days"),
-            confirmLabel: $l("Add"),
-            type: "number",
-            pattern: "[0-9]*",
-            message: message(),
-            validate: async (value) => {
-                const parsedValue = Number.parseInt(value, 10);
-                if (!value || !parsedValue || Number.isNaN(parsedValue)) {
-                    throw $l("Please enter a valid number of days!");
-                }
-
-                return value;
-            },
-        });
-
-        if (daysToAdd < 0) {
-            this._removeExpiration();
-        } else if (daysToAdd > 0) {
-            this._renewItemExpiration(daysToAdd);
+    private async _addExpiresAfter() {
+        if (this._editing) {
+            this._expiresAfter = 30;
+            await this.updateComplete;
+            this._expiresAfterInput.selectAll();
+        } else {
+            this.edit("addExpiration");
```

Comment:
Given the new approach, I think this UX is a bit nicer and more consistent with the rest of the editing experience.